components/host-dashboard/AddFundsModal.js

```diff
@@ -352,6 +352,11 @@ const AddFundsModal = ({ host, collective, ...props }) => {
               source: resultOrder.fromAccount,
               tier: resultOrder.tier,
             });
+            formik.setValues({
+              amount: values.amount,
+              fromAccount: resultOrder.fromAccount,
+              description: resultOrder.description,
+            });
```

Comment:
Please a code comment explaining why you're doing this. Precise about why those 3 values  `amount`, `fromAccount` and `description` and not anything else

Reply:
I've added a comment; let me know if that looks good. Thanks.